src/util/mapbox.js

```diff
@@ -178,7 +178,7 @@ export class RequestManager {
         if (accessToken[0] === 's')
             throw new Error(`Use a public access token (pk.*) with Mapbox GL, not a secret access token (sk.*). ${help}`);
 
-        urlObject.params = urlObject.params.filter((d) => !d.includes('access_token'));
+        urlObject.params = urlObject.params.filter((d) => d.indexOf('access_token') !== -1);
```

Comment:
Invert the comparison — "not includes" will translate to `indexOf(...) === -1`.

Reply:
Thank you Vlad. Classic.